### ISSUE TITLE
The  autocomplete for setting up a java Debugger is incomplete

### ISSUE THREAD

**Issue by musienko-maxim** [eclipse-che/che#15183]
### Describe the bug
After invocation the autocomplete in `launch.json` configuration file, the autocomplete list is incomplete. It influence Happy Path test.
### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Steps to reproduce

* Debug-> Open configuration
* Set cursor into configuration array 
* Invoke autocomplete
**Expected result:** we should get  get autocomplete list like on screenshot
![Selection_010](https://user-images.githubusercontent.com/1640058/68775206-ce750e80-0636-11ea-8a9a-a8251e2aa5de.png)

**Actual result:** the autocomplete list is inconsistent
![Selection_013](https://user-images.githubusercontent.com/1640058/68780314-c7ea9500-063e-11ea-8faa-a3696922cd38.png)

### Runtime
   - [ ] kubernetes (include output of `kubectl version`) 
   - [ ] Openshift (include output of `oc version`)
   - [x ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl 
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer 
       - [ ] Windows
       - [x] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify) 
   - [ ] other: please specify

**Comment by benoitf** [comment]
Hello @musienko-maxim it seems the 'Describe the bug' section is missing some part ?

**Comment by musienko-maxim** [comment]
@benoitf  Thx for remark, updated

**Comment by tolusha** [comment]
@musienko-maxim 
Have you tried to reproduce the issue on pure Theia?
If so, pls create an issue in corresponding repository.

**Comment by RomanNikitenko** [comment]
I think it can be caused by https://github.com/eclipse-theia/theia/pull/6255
I created the corresponding issue for `tasks.json` file.
Please see more details [here](https://github.com/eclipse-theia/theia/issues/6542)

**Comment by l0rd** [comment]
Set as a blocker because this issue affects our Happy Path PR check

**Comment by tsmaeder** [comment]
@musienko-maxim can we get the relevant logs, please?

**Comment by musienko-maxim** [comment]
@tsmaeder Sorry for huge delaying i forgot track this issue was very busy with CRW as mentioned  you can obrain the logs from the job:
`https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Che-Theia-PR-check-E2E-Happy-path-tests-against-K8S__reserved/573/artifact/logs-and-configs/workspace-logs/vscode-javai5u-container.log/*view*/`
If it is not be enough please ping me i will try to provide it directly

**Comment by tsmaeder** [comment]
Ok, there is no code completion at all for both launch.json nor tasks.json. I just tested this in a workspace using typescript (che nightly). This is definitely not specific to Java or any other language. 
There is nothing in browser or workspace logs that seems in any way relevant to the problem. It just looks like no code completion is set up for launch.json.
@tolusha you're worked with launch.json code completion before; could you have a look?

**Comment by tolusha** [comment]
@musienko-maxim @dmytro-ndp 
Could you confirm that it works in local Theia?

**Comment by musienko-maxim** [comment]
@tolusha Yes, i've just built  Theia and got something like this:
![Selection_014](https://user-images.githubusercontent.com/1640058/69246820-1acdca80-0bb2-11ea-9121-4639467147d7.png)
![Selection_015](https://user-images.githubusercontent.com/1640058/69246826-1d302480-0bb2-11ea-8a5f-13b53e22665c.png)

**Comment by tsmaeder** [comment]
This is a duplicate of https://github.com/eclipse-theia/theia/issues/6542 where the json language server seems broken, probably due to the update of our monaco version.

**Comment by l0rd** [comment]
This issue is considered a blocker because it makes the PR check fail. The JSON lang server not working is an issue but not a blocker.

To address the PR check problem @musienko-maxim we can editing the `launch.json` at all. The debug config can be setup in the devfile like we do it [here](https://github.com/eclipse/che-devfile-registry/blob/master/devfiles/java-web-spring/devfile.yaml#L54). That's also the preferred way to do that for our users.

**Comment by rhopp** [comment]
I've created https://github.com/eclipse/che/issues/15297 to unblock PR check tests (I've set it as Blocker severity, so we can lower the severity of this one to P1)

**Comment by tsmaeder** [comment]
FYI: @tolusha has provided an updated fix for the issue. With a bit of luck, we can merge the fix later toda.

**Comment by tolusha** [comment]
@benoitf 
The corresponding PR has been merged

**Comment by benoitf** [comment]
@tolusha nothing to do for che-theia as it's using master branch of theia so code is there once it's merged upstream.

but we might face now the issues about webviews

**Comment by tolusha** [comment]
@benoitf 
How is it related to PR?

**Comment by benoitf** [comment]
@tolusha because all PR reviews were red

If you need a fresh image of che-theia, it just needs a new commit or a re-spin of the build
